qa/rpc-tests/test_framework/util.py

```diff
@@ -548,7 +548,4 @@ def create_lots_of_big_transactions(node, txouts, utxos, fee):
 
 def get_bip9_status(node, key):
     info = node.getblockchaininfo()
-    for row in info['bip9_softforks']:
-        if row['id'] == key:
-            return row
-    raise IndexError ('key:"%s" not found' % key)
+    return info['bip9_softforks'][key]
```

Comment:
@MarcoFalke even better. The point is to preserve the previous behaviour of throwing a meaningful exception rather than leaving it to Python which results in `Unexpected exception caught during testing: 'xxxx'`.